src/app.tsx

```diff
@@ -15,13 +14,40 @@ import { ScheduleDataActionCreator } from "./state/reducers/month-state/schedule
 import { NotificationProvider } from "./components/common-components/notification/notification.context";
 import { NetlifyProFooter } from "./components/common-components/netlify-pro-footer/netlify-pro-footer.component";
 import isElectron from "is-electron";
+import { Box } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import JiraLikeDrawer from "./components/common-components/drawer/jira-like-drawer.component";
+import { JiraLikeDrawerProvider } from "./components/common-components/drawer/jira-like-drawer-context";
+import ScssVars from "./assets/styles/styles/custom/_variables.module.scss";
 
 interface TabData {
   label: string;
   component: JSX.Element;
 }
 
+const useStyles = makeStyles(() => ({
+  root: {
+    display: "flex",
+  },
+  content: {
+    display: "block",
+    overflowX: "auto",
+    overflowY: "auto",
+    height: `calc(100vh - ${parseInt(ScssVars.footerHeight.slice(0, -2))}px)`,
+    flexGrow: 1,
+  },
+  drawer: {
+    marginTop: ScssVars.headerHeight,
+    background: ScssVars.white,
+    borderLeft: "1px solid #EFF0F6",
```

Comment:
Może te kolorki też do zmiennych? XD Czuję się jak jakiś "człowiek zmienna", ale jakoś to mi się bardzo rzuca w oczy

Reply:
Jak wyszukuję cmd+shift+f w całości projektu ten kolor to się nie pojawia, nie wydaje mi się sensowne tworzenie koloru "drawerBorderLeftColor", bo dojdziemy do skrajności, gdzie będziemy tworzyć zmienne typu "padding10, padding11, padding12 etc"

Reply:
borderTop: "1px solid #E9EEF9"` - netlify-pro-footer.component.tsx
borderLeft: "1px solid #EFF0F6", - app.tsx
Chodziło mi o to, żeby stworzyć jeden borderColor, bo jak widać używamy bardzo podobnych w różnych miejscach. Nie jest to jakos specjalnie ważne, ale zawsze to większy porządek.

Reply:
Tylko pytanie który użyć i czy będzie pasowało. Trochę "nie mi tu oceniać". Można Adze powiedzieć, żeby ponazywała paletę kolorów i tylko ich używała, to by było idealnie